### DISCUSSION TITLE
Support for python ?

### DISCUSSION THREAD

**Discussion by jkc20** [xournalpp/xournalpp#2559] [Ideas]
Hi, I would like to use python for plugins.
Is there any reason that you build your own plugin-system and don't use:
https://wiki.gnome.org/Projects/Libpeas
in the latter case it would be quiet easy to extend it with pyhton.
Tx
Jürgen

**Comment by rolandlo** [comment]
Libpeas does look like a nice plugin system. It would have several advantages over our own system. Probably only the creator @andreasb242 of our plugin system can tell you, why he chose to implement his own system from scratch. If you have experience in implementing libpeas into an application like ours, feel free to provide some details on how that would have to be done (without loosing the current functionality provided by our [plugin API](https://xournalpp.github.io/guide/plugins/#plugin-api))
